emscripten/buildToolkit

```diff
@@ -34,7 +34,7 @@ my $VEROVIO_ROOT  = "..";
 # Memory is increased (TOTAL_STACK) for processing large files (tested up to 7MB)
 # Empirically, the memory amount required is roughly 5 times the file size.
 # This can be disabled in the light version, which uses the default memory settings.
-my $FLAGS = "-O3";
+my $FLAGS = "-Os";
```

Comment:
We typically don't alter the build script in the commit even though we do to build the toolkit.